syncd/FlexCounter.cpp

```diff
@@ -1588,6 +1689,51 @@ void FlexCounter::collectBufferPoolCounters(
     }
 }
 
+void FlexCounter::collectTunnelCounters(
+        _In_ swss::Table &countersTable)
+{
+    SWSS_LOG_ENTER();
+
+    // Collect stats for every registered tunnel
+    for (const auto &kv: m_tunnelCounterIdsMap)
+    {
+        const auto &tunnelVid = kv.first;
+        const auto &tunnelId = kv.second->tunnelId;
+        const auto &tunnelCounterIds = kv.second->tunnelCounterIds;
+
+        std::vector<uint64_t> tunnelStats(tunnelCounterIds.size());
+
+        // Get rif stats
```

Comment:
I think here is a typo, should be ```tunnel``` but not ```rif```

Reply:
Modified